### PR TITLE
Add missing steps to README for creating elasticsearch index

### PR DESCRIPTION
**Pull request by thearifismail**
# Overview

This PR is being created to address [RHINENG-6138](https://issues.redhat.com/browse/RHINENG-6138).

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by jpramos123 — DISMISSED** [review]
LGTM!

Thanks

File: docs/connect_to_k8s_env.md [hunk 1]
```diff
@@ -14,10 +14,23 @@ Debugging Host Inventory code requires a database, kakfa broker, Kafka Zookeeper
     ```bash
         docker/podmin login -u <usrename> quay.io
     ```
-3. Clone the [xjoin-operator](https://github.com/RedHatInsights/xjoin-operator) code repository and `cd` into the `xjoin-operator` directory and run:
+3. Clone the [xjoin-operator](https://github.com/RedHatInsights/xjoin-operator) code repository and `cd` into the `xjoin-operator` directory.
+4. Deplopy host-inventory:
```
Comment (kruai) [inline docs/connect_to_k8s_env.md:18]:
```suggestion
4. Deploy host-inventory:
```

**Review by kruai — CHANGES_REQUESTED** [review]
There's a typo, but it looks good other than that

**Review by kruai — APPROVED** [review]

**Pull request merged**